settings.yaml

```diff
@@ -49,3 +49,4 @@ sagemaker:
 
 openai:
   api_key: ${OPENAI_API_KEY:}
+  model: gpt-4
```

Comment:
I'd default to "gpt-3.5-turbo" in this settings.yaml, given it is the one being used by most people.